packages/core/src/hydration/annotate.ts

```diff
@@ -161,13 +166,15 @@ function annotateLContainerForHydration(lContainer: LContainer, context: Hydrati
  * @param appRef An instance of an ApplicationRef.
  * @param doc A reference to the current Document instance.
  */
-export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
+export function annotateForHydration(appId: string, appRef: ApplicationRef, doc: Document) {
```

Comment:
Just realized that we don't need to pass `appId` here, since we can retrieve it from the injector available on appRef (see another comment below).
```suggestion
export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
```

Reply:
I tried this earlier and I got a circular dependency build failure

Reply:
done